az-digital/az_quickstart
# Closes #1207 - Change card breakpoints to match Bootstrap 4 (break sooner)

## Description
Modifies the card deck break points to match Bootstrap 4. The breakpoints were all bumped up a level in the transition from Bootstrap 3 --> 4, so `-md` should now be `-lg`, and `-sm` should be `-md`. This will help alleviate the issue of cards becoming overly squished at the smaller end of the current "desktop" breakpoint. 

Review site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1526.probo.build/

## Related issues
Closes #1207 

## How to test new content (Probo)
- Add some cards to a Page and play with the breakpoints on desktop, tablet, and mobile. 
- Add a sidebar to that same page to ensure it looks reasonable when a sidebar is included.

## How to test existing content (database update on existing site)
- Add the PR diff file as a patch to a local/existing/Pantheon/etc build.
- Run the database update
- Ensure breakpoint selections for card decks update accordingly

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
